configurations/es5.js

```diff
@@ -0,0 +1,25 @@
+module.exports = {
+    extends: [
+        '../rules/eslint/best-practices/on.js',
+        '../rules/eslint/possible-errors/on.js',
+        '../rules/eslint/ecma-script-6/off.js',
+        '../rules/eslint/node-js-and-common-js/off.js',
+        '../rules/eslint/strict-mode/on.js',
+        '../rules/eslint/stylistic-issues/on.js',
+        '../rules/eslint/variables/on.js',
+        '../rules/eslint/deprecated/off.js'
```

Comment:
Ahh, ok, I see now. Can we add a comment or a todo to these, so they are describing why they are there?